test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -35,13 +35,52 @@ internal static class RegressionTest
 
         private static int Main(string[] args)
         {
-            return Parser.Default.ParseArguments<Options>(args).MapResult(
-                Run,
-                _ =>
+            if (args.Length > 1 && args[0].Equals("warm-up"))
```

Comment:
Add `WarnUp` to `Options`, don't manually parse command line args.

Reply:
Done.